src/ownership-verifiers/twitterV2.test.ts

```diff
@@ -107,3 +107,16 @@ test("should be able to take only the good access token", async () => {
     "test_twitter_code_ok"
   );
 });
+
+test("should throw error when token queue is empty", async () => {
+  expect(await fifoQueue.isEmpty()).toBeTruthy()
+  await expect(twitterV2OwnershipVerifierTest.getAccessToken()).rejects.toThrowError("No tokens in the pool");
+});
+
+test("should throw error when token queue is empty after reaching max refresh threshold", async() => {
```

Comment:
Are you sure to test the whole algo? I would have also created a test where I do a loop over 10 (threshold) access token to be sure it's removed from the queue